docs/build/api/get_offerings/get_offerings.md

```diff
@@ -0,0 +1,539 @@
+---
+sidebar_position: 2
+---
+
+# Finding compute resources on the Marketplace
+
+The Fluence compute marketplace is a decentralized platform where you can find and rent compute resources from various providers worldwide. Each provider has its own offers in different data centers with different configurations and prices. The marketplace API makes it easy to discover resources that meet your specific requirements.
+
+In this guide, we'll walk through how to:
+
+1. Search for available compute offers
+2. Filter resources based on your requirements
+3. Understand and compare different provider offerings
+4. Select the optimal resources for your needs
+
+## Searching for Available Offers
+
+The marketplace offers a powerful API endpoint that allows you to search for available compute resources with specific filters:
+
+```bash
+POST https://api.fluence.dev/marketplace/offers
+```
+
+### Understanding search parameters (constraints)
+
+You can use the request body to filter offers based on your specific requirements. All filters are optional, you can use them to find whether there are offers that match your requirements.
+
+**Example of a request body with all filters applied:**
+
+```json
+{
+  "basicConfiguration": "cpu-8-ram-16gb-storage-25gb",
+  "additionalResources": {
+    "ramMb": 2048,
+    "storage": [
+      {
+        "type": "NVMe",
+        "megabytes": 20480
+      }
+    ]
+  },
+  "hardware": {
+    "cpu": [
+      {
+        "manufacturer": "AMD",
+        "architecture": "Zen" // TODO: add brand and generation
+      }
+    ],
+    "memory": [
+      {
+        "type": "DDR",
+        "generation": "5"
+      }
+    ],
+    "storage": [
+      {
+        "type": "NVMe"
+      }
+    ]
+  },
+  "datacenter": {
+    "countries": ["FR"] // TODO: maybe add requirements for data center certifications
+  },
+  "maxTotalPricePerEpochUsd": "12.57426"
+  // TODO: add amount of instances (with additional resources)
+}
+```
+
+Let's break down the request body parameters and their usage
+
+#### Basic Configuration
+
+A predefined configuration string that specifies a common resource profile. Use this for quick filtering without specifying individual resources
+
+**Example:**
+
+```json
+{
+  "basicConfiguration": "cpu-8-ram-16gb-storage-25gb"
+}
+```
+
+This filter would show offers with 8 CPU cores, 16GB RAM, and 25GB storage.
+
+To get the list with names of all available basic configurations on the marketplace, you can use specific endpoint. Read more about it in the [Basic configurations available on the Marketplace](#basic-configurations-available-on-the-marketplace) section.
```

Comment:
i don't see the GET /marketplace/basic_configurations this in swagger at all. i only see /offers and with a very limited set of actions